src/shared/components/login/socialLogin.js

```diff
@@ -0,0 +1,172 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import config from 'config/environment';
+import Section from 'shared/components/section/section';
+import Form from 'shared/components/form/form';
+import FormZipCode from 'shared/components/form/formZipCode/formZipCode';
+import FormPassword from 'shared/components/form/formPassword/formPassword';
+import FormButton from 'shared/components/form/formButton/formButton';
+import styles from 'scenes/home/informationForm/informationForm.css';
+import _ from 'lodash';
+import * as CookieHelpers from '../../utils/cookieHelper';
+
+class SocialLogin extends Component {
+  state = {
+    zip: '',
+    password: '',
+    error: false,
+    isLoading: false
+  }
+
+  componentWillUnmount() {
+    this.onExit();
+  }
+
+  onZipChange = (value, valid) => {
+    this.setState({ zip: value, zipValid: valid });
+  }
+
+  onPasswordChange = (value, valid) => {
+    this.setState({ password: value, passwordValid: valid });
+  }
+
+  onExit = () => {
+    window.localStorage.removeItem('firstname');
+    window.localStorage.removeItem('lastname');
+    window.localStorage.removeItem('email');
+    console.log(window.localStorage.getItem('email'));
+  }
+
+  run = (First, Last, Email) => {
+    axios
+      .post(`${config.backendUrl}/users/exist`, {
+        // data: code
+        user: {
+          email: Email
+        }
+      })
+      .then(({ data }) => {
+        console.log('redirect is: ');
+        console.log(data.redirect_to);
+        window.localStorage.setItem('firstname', `${First}`);
+        window.localStorage.setItem('lastname', `${Last}`);
+        window.localStorage.setItem('email', `${Email}`);
+        if (data.redirect_to === '/social_login') {
+          console.log('rendering');
+          window.location = data.redirect_to;
```

Comment:
Consider utilizing React Router to not lose history of the session:
https://stackoverflow.com/questions/42123261/programmatically-navigate-using-react-router-v4

I believe method 1 or 3 will work in this situation.